src/extensions/volumes-and-segmentations/index.ts

```diff
@@ -19,7 +19,7 @@ import { VolsegEntryFromRoot, VolsegGlobalStateFromRoot, VolsegStateFromEntry }
 import { VolsegUI } from './ui';
 
 
-const DEBUGGING = window.location.hostname === 'localhost';
+const DEBUGGING = window?.location?.hostname === 'localhost';
```

Comment:
This one might not be enough. Maybe `type window !== 'undefined' && window !== null ? window.location?.hostname === 'localhost' : false` ?

Reply:
yeah was just going to add a note/update this - `window` will be undefined if it's not polyfilled